OracleWebCenterSites/README.md

```diff
@@ -1,461 +1,45 @@
-##### 4. How to modify start/stop admin/managed server scripts?
-You can find these scripts here: dockerfiles/12.2.1.3/sites-container-scripts) sites-container-scripts are located at `../docker-images/OracleWebCenterSites/dockerfiles/12.2.1.3/sites-container-scripts/*` 
-
-##### 5. Why do I get an error message as "... RCU exists already"?
-Most likely, you're not running this command for the first time. The RCU_prefix may be present already. Drop the corresponding schemas or use a different prefix.
-
-##### 6. Where can I find RCU configuration Wizard and WebCenter Sites configuration scripts?
-See [Readme.md](dockerfiles/12.2.1.3/wcs-wls-docker-install/README.md) located at `../docker-images/OracleWebCenterSites/dockerfiles/12.2.1.3/wcs-wls-docker-install/README.md`
+For pre-built images containing Oracle software, please check the [Oracle Container Registry](https://container-registry.oracle.com).
 
-##### 7. How do I configure WebCenter Sites with an On-Prem Oracle Database instance? 
-Set DB_CONNECTSTRING connection string parameter as mentioned in section [Update the environment file](#1-update-the-environment-file-1).
+## Building the WebCenter Sites Docker images
+You must download the WebCenter Sites binary and put it in its correct location (see `.download` files inside `dockerfiles/<version>`).
 
-##### 8. Alternate download location for Oracle Fusion Middleware Infrastructure and Oracle Database Images? 
-Before you build an Oracle WebCenter Sites image, download the Oracle Fusion Middleware infrastructure and Oracle Database images from the [Docker Store.](https://store.docker.com/)
+Before you build, select the version and distribution for which you want to build an image, then download the required packages (see `.download` files) and place them in the folder of your distribution version of choice. Then, from the `dockerfiles` folder, run the `buildDockerImage.sh` script.
 
-If you download Oracle Fusion Middleware infrastructure from Docker Store, then retag using below command:
-```
-   $ docker tag store/oracle/fmw-infrastructure:12.2.1.3 oracle/fmw-infrastructure:12.2.1.3
-```
-If you download Oracle Database from Docker Store, then retag using below command:
-```
-   $ docker tag store/oracle/database-enterprise:12.2.0.1 database/enterprise:12.2.0.1
-```
-##### 9. How do I build an Oracle Database 12.2.1.x base image?
-If you want to build your own Oracle Database image, use the Docker files and scripts in the [Oracle Database](../OracleDatabase) GitHub repository.
+		-bash-4.2$ sh buildDockerImage.sh -h
+		
+		Usage: buildDockerImage.sh -v [version] [-s] [-c]
+		Builds a Docker Image for Oracle WebCenter Sites.
+		
+		Parameters:
+				-v: version to build. Required.
+						Choose one of: 12.2.1.3  12.2.1.4
+				-c: enables Docker image layer cache during build
+				-s: skips the MD5 check of packages
+		
+		LICENSE CDDL 1.0 + GPL 2.0
```

Comment:
Where does this come from? We're definitely not licensed under CDDL 1.0 + GPL 2.0. This line should not be here.